megamek/src/megamek/client/ui/swing/boardview/BoardView1.java

```diff
@@ -1050,6 +1050,28 @@ public void performAction() {
                         toggleKeybindsOverlay();
                     }
                 });
+        
+        // Register the action for TOGGLE_HEX_COORDS
+        controller.registerCommandAction(KeyCommandBind.TOGGLE_HEX_COORDS.cmd,
+                new CommandAction() {
+
+                    @Override
+                    public boolean shouldPerformAction() {
+                        if (shouldIgnoreKeyCommands()) {
+                            return false;
+                        } else {
+                            return true;
+                        }
```

Comment:
```suggestion
                        return !shouldIgnoreKeyCommands();
```

Reply:
Oh indeed :) Never noticed. I'll correct this for all the other binds too.